src/electionguard/tracker.py

```diff
@@ -39,32 +39,29 @@ def tracker_hash_to_words(
     :return: Human readable tracker string or None
     """
 
-    segments = q_to_bytes(tracker_hash)
+    segments = tracker_hash.to_bytes()
     words: List[str] = []
-    for value in segments:
-        word = get_word(value)
-        if word is None:
-            return None
-        words.append(word)
-    # FIXME ISSUE #82 Minimize length of tracker
-    return seperator.join(words)
+    for i in range(0, len(segments), 4):
+        # Select 4 bytes for the segment
+        first = segments[i]
+        second = segments[i + 1]
+        third = segments[i + 2]
+        fourth = segments[i + 3]
 
+        # word is byte(1) + 1/2 of byte(2)
+        word_part = get_word((first * 16 + (second >> 4)))
 
-def tracker_words_to_hash(
```

Comment:
The `electionguard-web-api` currently exposes this. Is there no case where a dev may search by the tracker_code but lookup by tracker_hash? Or do we want to add a ticket for someone to create this function for `hacktoberfest`

CC @rkorsak

Reply:
It's not used anywhere in the protocol and specifically it is not used in generating any cryptographic values.  i think it makes sense to discourage using the tracking code anywhere except as a human-readable lookup value in the published results.  further, this representation is entirely arbitrary.  My vote is to remove the endpoint from the web api

Reply:
Fair point. Just denoting it.